ofrak_core/ofrak/gui/server.py

```diff
@@ -205,11 +231,18 @@ async def create_root_resource(self, request: Request) -> Response:
         root_resource = await self._ofrak_context.create_root_resource(name, resource_data, (File,))
         if request.remote is not None:
             self._job_ids[request.remote] = root_resource.get_job_id()
+
+        script_str = rf"""
+        with open("{name}", "rb") as fh:
+            resource_data = fh.read()
+        root_resource = await ofrak_context.create_root_resource_from_file("{name}")"""
```

Comment:
```suggestion
        root_resource = await ofrak_context.create_root_resource_from_file("{name}")"""
```
`resource_data` is unused, don't need to open and read the file if we are going to use `create_root_resource_from_file`